src/plugins/vis_type_timeline/server/series_functions/graphite_helper.test.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import * as helper from './graphite_helper';
```

Comment:
let's avoid `import *`

refer to https://github.com/airbnb/javascript#modules--no-wildcard

import only your need.

`import  { isValidConfig } from './graphite_helper'`

Reply:
I see your concerns. For the unit test, I have `isValidConfig ( call isBlockedURL)` . I am trying to test `isValidConfig` will  call `isBlockedURL` when a blockedlist is passed. So I import all as a module. Then mock one function in this module.  I didn't any other better way for testing it. There is a long [discussion](https://github.com/facebook/jest/issues/936) for testing function inside another function. I can investigate more to see other ways afterwards.